test/mako/alerter/slack/message.go

```diff
@@ -21,26 +21,35 @@ import (
 	"sync"
 	"time"
 
-	"knative.dev/pkg/test/mako/alerter"
+	"knative.dev/pkg/test/mako/alerter/shared"
 	"knative.dev/pkg/test/slackutil"
 )
 
-const messageTemplate = `
+const (
+	// do not send alert on the same channel within 24 hours
+	minInterval     = 24 * time.Hour
```

Comment:
Flag?

Reply:
I'm defining the `Alerter` as a library instead of a binary, are you suggesting passing it as a parameter in `Setup`?

Reply:
pass it as var in SendAlert()? I dont think it is needed in setup

Reply:
I feel it's something that should be defined by the framework, rather than the invokers. It's a bit strange if the caller needs to specify the `minInterval` every time it calls `SendAlert`.

Reply:
No I actually mean flag. When you launch the job you can give it 24 you can give it 36 you can give it 6. This way you need recompile re-deploy

Reply:
We'll have to call these functions from the benchmark, since it needs the benchmarking result from `q.Store()`. How about we pass the flag to the benchmark container, Like the line below?
https://github.com/knative/serving/blob/8165ebf018a26b259348c5b7fea81860fd084591/test/performance/load-test/load-test.yaml#L64

Then we'll need to define it as an arg in `Setup`.

Reply:
It's not the best practice, but you can define your flag right here, right in this file.

Reply:
Done.